templates/web/fixmystreet.com/next_steps.html

```diff
@@ -1,11 +1,14 @@
 [% IF utm_content == 'questionnaire completed' %]
-<div style="border: solid 2px black; margin: 0 auto; padding: 1em; width: 90%; margin-bottom: 1em">
-    <h2 style="margin-top:0">Can you help us out with a quick survey?</h2>
-    <p>We want to know more about who uses FixMyStreet and what you use it for.
-    <p>Can you spare 5 minutes for a quick survey?
-    <p>There is the opportunity to enter a draw for a voucher of your choice (but also it would just help us out)
-    <p style="margin-bottom:0">
-    <a href="https://survey.alchemer.com/s3/6214003/FixMyStreet-User-Survey?[% c.cobrand.user_survey_information %]" class="btn btn--primary">I want to fill out the user survey</a>
+<div class="survey-cta">
+  <div class="survey-cta__content">
+    <p class="content__preheading">Can you spare 5 minutes for a survey about FixMyStreet?</p>
+    <h2 class="content__heading">We want to know more about who uses FixMyStreet and what you use it for to help us improve the service.</h2>
+    <p>As a thank you, there's the opportunity to enter a prize draw for a voucher of your choice.</p>
+    <a href="https://survey.alchemer.com/s3/6214003/FixMyStreet-User-Survey?[% c.cobrand.user_survey_information %]" class="btn btn--secondary">Complete the short survey</a>
+  </div>
+  <div class="survey-cta__img">
+    <img src="/cobrands/fixmystreet.com/images/survey-img-1.svg" alt="Illustration: User answering survey" width="300" height="300">
```

Comment:
The alt attribute is an alternative for the image, not a description. I think this image is purely decorative and so can have a blank alt attribute.

Reply:
Thanks @dracos =) I still have the file opened so I got rid of the alt attribute.
Let me know if there is anything else

Reply:
No, you can't get rid of it, because that will cause screen readers to read the file path or similar. If an image is purely decorative, the correct output is `alt=""` to show that you have deliberately given it that alt attribute, that's what I meant by "blank", not missing. Like I said, I can change it as part of the merge, don't worry about it.

Reply:
Thanks I didn't know that, thanks for merging.